contents/handbook/growth/marketing/value-propositions.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Value propositions
+sidebar: Handbook
+showTitle: true
+---
+
+# Value propositions
+
+Value propositions are the promises we make to our prospective customers about what PostHog does and the value it offers to them.
+
+Value propositions are not intended to be used verbatim, but to be interpreted, referred to and expanded upon. They're useful for checking and creating paid ads or for writing site copy, for example.
+
+PostHog has several value propositions. Each is structured like a story: 
+
+- The **proposition** we make (e.g. _Have more fun at the arcade_)
+- The **challenge** before PostHog (e.g. _Most arcade games are for one player_)
+- The **experience** of this challenge (e.g. _It's not fun to play on your own_)
+- The **solution** PostHog offers (e.g. _Our arcade game has a two-player mode_) 
+- The **result** of the solution (e.g. _You'll have twice as much fun_)
+
+Our value propositions also call on our messaging pillars, which map to specific functionality that reinforce the proposition (e.g. _full-size controls for player two_). 
+
+### 1. An all-in-one platform
+
+PostHog offers a suite of product analysis tools, including funnels, heat maps, session recording and more, all in one easy-to-use platform. This enables data and engineering teams to get information faster, without writing any SQL, while less technical teams are also able to self-serve and get answers without needing support.
+
+- **Proposition:** Get all the analysis you need in one place, without writing any SQL.
+- **Challenge:** You have to go between many different tools, often writing cumbersome SQL or making requests of other teams to get information.
+- **Experience:** You aren't able to progress quickly and even simple requests can take a lot of time.
+- **Solution:** PostHog is the only product analytics solution that combines tools such as funnels, session recording and feature flags into a single self-serve platform.
+- **Result:** Now you can get answers faster and focus on improving your product, rather than writing SQL.
```

Comment:
I'd probably remove 'rather than writing SQL' here as well - mentioning SQL twice I think is enough.